data/assets/examples/tileprovider/tileproviderbyindex/example.asset

```diff
@@ -0,0 +1,58 @@
+-- This example file adds a layer to a globe that has a base layer and then replaces one
+-- hemisphere of the planet with a single image. Recommended reading for this example is
+-- the documentation on the DefaultTileProvider.
+
+-- Download some example images that we can use
+local images = asset.resource({
+  Name = "Earth Textures",
+  Type = "HttpSynchronization",
+  Identifier = "earth_textures",
+  Version = 3
+})
+
+-- Define the TileProvider
+local TileProvider = {
+  Identifier = "Example",
+  Type = "TileProviderByIndex",
+  Enabled = true,
+  -- The default tile provider that is used for the whole globe
+  DefaultTileProvider = {
+    Identifier = "Blue_Marble",
+    FilePath = images .. "earth_bluemarble.jpg"
+  },
+  TileProviders = {
+    -- We only define one additional tile provider that overwrites the right
+    -- hemisphere of the globe
+    {
+      Index = { X = 0, Y = 0, Level = 2 },
+      TileProvider = {
+        Identifier = "Blue_Marble_Night",
+        FilePath = images .. "earth_night.png"
+      }
+    }
+  }
+}
+
+-- Define the scene graph node
+local Node = {
+  Identifier = "TileProviderByIndex_Example",
+  Renderable = {
+    Type = "RenderableGlobe",
+    Layers = {
+      -- The globe has exactly one layer, which is the one we defined above
+      ColorLayers = { TileProvider }
+    }
+  },
+  GUI = {
+    Name = "Minimal",
+    Path = "/Examples/TileProviderByIndex"
+  }
+}
+
+asset.onInitialize(function()
+  openspace.addSceneGraphNode(Node)
+end)
+
+asset.onDeinitialize(function()
+  openspace.removeSceneGraphNode(Node)
```

Comment:
I tried the example and I think it would be nice if you could go a bit closer to the globe, is that possible? Now the globe is max 20% of my vertical fov, I suppose because of the interaction sphere or something like that. It makes it a bit difficult to see the tiles 
![image](https://github.com/OpenSpace/OpenSpace/assets/24265984/8346e458-271a-46a7-8978-4e39c83b6b9b)